README.md

```diff
@@ -1 +1,187 @@
+onfido.applicant.find("applicant_id")                   # => Finds a single applicant
+onfido.applicant.list()                                 # => Returns all applicants
+```
+
+**Note:** Calling `onfido.applicant.destroy()` adds the applicant and all associated documents, photos, videos, checks, and reports to the deletion queue. They will be deleted 20 days after the request is made. An applicant that is scheduled for deletion can be restored but applicants that have been permanently deleted cannot.
+See https://documentation.onfido.com/#delete-applicant for more information.
+
+#### Documents
+
+Documents provide supporting evidence for Onfido checks.
+
+```java
+onfido.document.upload(inputStream, "fileName", requestBody) # => Creates a document
+onfido.document.find("document_id")                          # => Finds a document
+onfido.document.download("document_id")                      # => Downloads a document as a binary data
+onfido.document.list("applicant_id")                         # => Returns all applicant's documents
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Live Photos
+
+Live Photos, like documents, can provide supporting evidence for Onfido checks.
+
+```java
+onfido.live_photo.upload(inputStream, "fileName", requestBody) # => Creates a live photo
+onfido.live_photo.find("live_photo_id")                        # => Finds a live photo
+onfido.live_photo.download("live_photo_id")                    # => Downloads a live photo as binary data
+onfido.live_photo.list("applicant_id")                         # => Returns all applicant's live photos
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Checks
+
+Checks are requests for Onfido to check an applicant, by commissioning one or
+more "reports" on them.
+
+```java
+onfido.check.create("applicant_id", reportNamesArray) 
+onfido.check.find("check_id")
+onfido.check.resume("check_id")
+onfido.check.list("applicant_id")
+```
+
+#### Reports
+
+Reports provide details of the results of some part of a "check". They are
+created when a check is created, so the Onfido API only provides support for
+finding and listing them. For paused reports specifically, additional support for resuming and
+ cancelling reports is also available.
+
+```java
+onfido.report.find("report_id")
+onfido.report.list("check_id")
+onfido.report.resume("report_id")
+onfido.report.cancel("report_id")
+```
+
+#### Address Lookups
+
+Onfido provides an address lookup service, to help ensure well-formatted
+addresses are provided when creating "applicants". To search for addresses
+by postcode, use:
+
+```java
+onfido.address.pick("SE1 4NG")
+```
+
+#### Webhook Endpoints
```

Comment:
I think we should remove this section (h4: "Webhook Endpoints") as we have a "Webhooks" section below. I'm not sure it's too important to say in this README to say people can see webhooks they've set up in the Dashboard.